src/logger/fileWriter.c

```diff
@@ -383,6 +390,9 @@ TESTABLE_STATIC int logging_sample(struct logging_status *ls,
                 if (WRITING_ACTIVE != ls->writing_status)
                         open_log_file(ls);
 
+                if (SD_CARD_NOT_PRESENT == ls->writing_status)
```

Comment:
Ah, I remember. open_log_file(ls) will modify the struct, so that's why we check afterwards. So, I'll leave it as-is.